resources/core/charts/logging/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: logging
+version: 0.0.1
```

Comment:
in the resources/core/README.md there is an information that in case of adding new subchart you need to update requirements.yaml

Reply:
added